lib/index.js

```diff
@@ -1,19 +1,48 @@
 // This file is in the entry point in your webpack config.
 import './styles.scss';
 import CurrentWeather from './current-weather'
+import BackgroundImage from './background-image'
 
-$('form').on('submit', function(event) {
+$(document).ready(function() {
+  $('#location-form').on('submit', getLocationInfo);
+  $('#sign-up').on('submit', registerUser);
+})
+
+const registerUser = (event) => {
+  event.preventDefault();
+  $('.errors').text('');
+  const email = $('input')[0].value;
+  const password = $('input')[1].value;
+  const confPassword = $('input')[2].value;
+  if(!password || !confPassword || password !== confPassword) {
+    displayError("Error: Please check passwords")
+  } else {
+    createUser(email, password, confPassword);
+  };
+};
+
+const createUser = (email, password, confPassword) => {
+  const usersUrl = API_URL + "/users"
+  $.post(usersUrl, { email: email, password: password, password_confirmation: confPassword })
```

Comment:
If you have a key that matches the name of the existing variable in your code, you can simplify how you create the object to `{ email, password, password_confirmation: confPassword}`. Newer JS syntax allows this, just a nice way to reduce how much typing you have to do and it is more readable.

Reply:
NICE!! That's awesome.